manifests/dev/vsphere-7.0u2/guestcluster/1.19/pvcsi.yaml

```diff
@@ -345,6 +382,12 @@ spec:
       - name: device-dir
         hostPath:
           path: /dev
+      - name: sys-dir
+        hostPath:
+          path: /sys
+      - name: pvcsi-config-volume
+        configMap:
+          name: pvcsi-config
```

Comment:
Am i missing something here or should we be mounting the FSS configmap to CSI node rather than the pvcsi-config (which contains creds etc) ?

Reply:
As mentioned in our discussion before, the node needs to read the cns-csi conf to read the `InternalFeatureStates` and `FeatureStates` config values.